modules/stringtie/1.0/src/tagXSstrandedData.awk

```diff
@@ -0,0 +1,41 @@
+## Authored by Alex Dobin, available at https://github.com/alexdobin/STAR/tree/master/extras/scripts
```

Comment:
If this script is not modified, should we consider downloading it within the module, since this is not explicitly authored by us? If you are concerned about possible future script updates that would break the functionality, it is possible tor refer to a specific commit (instead of master) to ensure exact same version is always used

Reply:
I thought about this as well but doing it this way protects this working version of the script in case Alex Dobin one day decides to delete or modify this script from his repo.

Reply:
this makes sense, let's keep it as-is 😄